### PR TITLE
media-gfx/librecad: version bump 2.0.9

### PR DESCRIPTION
**Pull request by dxli**
version bump to the latest upstream stable

### CODE REVIEW

**Comment by monsieurp** [comment]
slis should be taking care of it but..

[21:16:48] [ @monsieurp ] !seen slis
[21:16:48] [ +willikins ] monsieurp: slis was last seen 7 months, 7 days, 8 hours, 19 minutes and 35 seconds ago, quitting IRC (Remote host closed the connection)

hence @gentoo/proxy-maint

File: media-gfx/librecad/librecad-2.0.9.ebuild [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+		eqmake5 -r
+	fi
+}
+
+src_install() {
+	dobin unix/librecad
+	use tools && dobin unix/ttf2lff
+	insinto /usr/share/${PN}
+	doins -r unix/resources/*
+	use doc && dohtml -r librecad/support/doc/*
```
Comment (jlec) [inline media-gfx/librecad/librecad-2.0.9.ebuild]: ```dohtml``` is deprecated, just use ```use doc && instinto html && dodoc -r librecad/support/doc/*```

File: media-gfx/librecad/librecad-2.0.9.ebuild [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+	insinto /usr/share/appdata
+	doins unix/appdata/librecad.appdata.xml
+	doicon librecad/res/main/"${PN}".png
```
Comment (jlec) [inline media-gfx/librecad/librecad-2.0.9.ebuild]: no need to quote ```${PN}```

File: media-gfx/librecad/librecad-2.0.9.ebuild [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+S="${WORKDIR}/LibreCAD-${PV}"
+
+src_prepare() {
+	# currently RS_VECTOR3D causes an internal compiler error on GCC-4.8
+	use 3d || sed -i -e '/RS_VECTOR2D/ s/^#//' librecad/src/src.pro || die
+}
+
+src_configure() {
+	if use qt4
+	then
```
Comment (jlec) [inline media-gfx/librecad/librecad-2.0.9.ebuild]: lease move this up one line. ebuild style is the following
```ebuild
if use xxx; then
    ....
else
    ....
fi
```

File: media-gfx/librecad/librecad-2.0.9.ebuild [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+	dev-libs/boost
+	dev-cpp/muParser
+	media-libs/freetype"
+
+RDEPEND="${DEPEND}"
+S="${WORKDIR}/LibreCAD-${PV}"
+
+src_prepare() {
+	# currently RS_VECTOR3D causes an internal compiler error on GCC-4.8
+	use 3d || sed -i -e '/RS_VECTOR2D/ s/^#//' librecad/src/src.pro || die
```
Comment (jlec) [inline media-gfx/librecad/librecad-2.0.9.ebuild]: better use proper ``ìf ... then```construct.

File: media-gfx/librecad/librecad-2.0.9.ebuild [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+
+IUSE="3d debug doc tools qt4 +qt5"
+REQUIRED_USE="|| ( qt4 qt5 )"
+
+DEPEND="
+	qt4? (
```
Comment (jlec) [inline media-gfx/librecad/librecad-2.0.9.ebuild]: Probably you should add SLOT operators like ```=``` to all those deps.

File: media-gfx/librecad/librecad-2.0.9.ebuild [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+	qt5? (
+		dev-qt/qtcore:5
+		dev-qt/qtgui:5
+		dev-qt/qthelp:5
+		dev-qt/qtprintsupport:5
+		dev-qt/qtsvg:5
+		dev-qt/qtwidgets:5
+		dev-qt/qtxml:5
+	)
+	dev-libs/boost
```
Comment (jlec) [inline media-gfx/librecad/librecad-2.0.9.ebuild]: slot operator missing.

File: media-gfx/librecad/librecad-2.0.9.ebuild (hunk 4 above)
Comment (idella) [inline media-gfx/librecad/librecad-2.0.9.ebuild]: yes, this innocent looking form goes wrong, and use of  ìf ... then`construct makes it "bullet proof"

File: media-gfx/librecad/librecad-2.0.9.ebuild (hunk 5 above)
Comment (kensington) [inline media-gfx/librecad/librecad-2.0.9.ebuild]: @jlec please don't suggest to add SLOT operators to Qt dependencies. As they do not explicitly define any subslot right now, the behaviour is undefined and will result in useless rebuilds. The next version of Qt will introduce a subslot but only to handle a specific use case.

Comment (jlec) [inline media-gfx/librecad/librecad-2.0.9.ebuild]: @kensington yes, I already figured that out. We had a chat in the proxy-maint irc channel and solved this. I probably should have left a message here, too.

**Comment by idella** [comment]
boost IS sub slotted so warrants addition of the slot operator

**Comment by idella** [comment]
dev-libs/boost

slot operator still missing otherwise ready to go

**Comment by dxli** [comment]
@idella since there's a slot for media-libs/freetype:2, it is added also

**Comment by monsieurp** [comment]
It's ready to go now, I guess.

**Pull request merged**